stdlib/influxdata/influxdb/monitor/check_test.flux

```diff
@@ -30,26 +29,23 @@ outData = "
 ,,1,000000000000000a,cpu threshold check,ok,statuses,whoa!,cpu,1527018820000000000,2018-05-22T19:54:20Z,threshold,vaaa,vbbb,cpu-total,host.local,90.62382797849732
 ,,2,000000000000000a,cpu threshold check,warn,statuses,whoa!,cpu,1527018860000000000,2018-05-22T19:54:20Z,threshold,vaaa,vbbb,cpu-total,host.local,7.05
 "
-
 data = {
     _check_id: "000000000000000a",
     _check_name: "cpu threshold check",
     _type: "threshold",
-    tags: {aaa: "vaaa", bbb: "vbbb"}
+    tags: {aaa: "vaaa", bbb: "vbbb"},
 }
-
-crit = (r) => (r.usage_idle < 5.0)
-warn = (r) => (r.usage_idle < 10.0)
-info = (r) => (r.usage_idle < 25.0)
-
+crit = (r) => r.usage_idle < 5.0
+warn = (r) => r.usage_idle < 10.0
+info = (r) => r.usage_idle < 25.0
 messageFn = (r) => "whoa!"
-
 t_check = (table=<-) => table
     |> range(start: -1m)
     |> filter(fn: (r) => r._measurement == "cpu")
     |> filter(fn: (r) => r._field == "usage_idle")
     |> filter(fn: (r) => r.cpu == "cpu-total")
-    |> v1.fieldsAsCols() // pivot data so there is a "usage_idle" column
+    |> v1.fieldsAsCols()
+    // pivot data so there is a "usage_idle" column
```

Comment:
This change is a little weird, since the comment refers to `fieldsAsCols` but appears after it in the re-formatted code. It should have an issue to fix it in my opinion.

Reply:
I have opened an issue with it. #3657